modules/tensor_mechanics/src/actions/TensorMechanicsAction.C

```diff
@@ -15,19 +15,21 @@ template<>
 InputParameters validParams<TensorMechanicsAction>()
 {
   InputParameters params = validParams<Action>();
-  params.addClassDescription("Set up stress divergence kernels");
+  params.addClassDescription("Set up stress divergence kernel");
   params.addParam<std::vector<NonlinearVariableName> >("displacements", "The nonlinear displacement variables for the problem");
-  params.addParam<NonlinearVariableName>("disp_x", "The x displacement");  // depricated
-  params.addParam<NonlinearVariableName>("disp_y", "The y displacement");  // depricated
-  params.addParam<NonlinearVariableName>("disp_z", "The z displacement");  // depricated
+  params.addParam<NonlinearVariableName>("disp_x", "The x displacement");  // deprecated
+  params.addParam<NonlinearVariableName>("disp_y", "The y displacement");  // deprecated
+  params.addParam<NonlinearVariableName>("disp_z", "The z displacement");  // deprecated
   params.addParam<NonlinearVariableName>("temp", "The temperature");
   params.addParam<std::string>("base_name", "Material property base name");
   params.addParam<bool>("use_displaced_mesh", false, "Whether to use displaced mesh in the kernels");
-
+  params.addParam<std::vector<SubdomainName> >("block", "The list of ids of the blocks (subdomain) that the stress divergence kernel will be applied to");
   params.addParam<std::vector<AuxVariableName> >("save_in_disp_x", "Auxiliary variables to save the x displacement residuals.");
```

Comment:
Why does this not follow the displacements vector coupling example? same for the diag stuff below